src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -154,11 +154,34 @@ public String getDocStringOrNothing() {
                 getDocString().getEscapedValue() +
                 "</div></div>";
     }
+    
+    private static int labelCnt = 0;
 
     private String formatError(String errorMessage) {
         String result = errorMessage;
         if (errorMessage != null && !errorMessage.isEmpty()) {
-            result = errorMessage.replaceAll("\\\\n", "<br/>");
+            if (errorMessage.startsWith( "<" )) {
```

Comment:
Change above to
``if (StringUtils.isNotEmpty(errorMessage)``

we should have apache common imported already